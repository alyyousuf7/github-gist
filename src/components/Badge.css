.badge {
    background: green;
    font-size: small;
    border-radius: 0.25em;
    padding: 2px;
}
