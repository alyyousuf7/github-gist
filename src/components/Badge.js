import PropTypes from 'prop-types';

import './Badge.css';

function Badge({ children }) {
  return <div className="badge">{children}</div>;
}

Badge.propTypes = {
  children: PropTypes.string.isRequired,
};

export default Badge;
