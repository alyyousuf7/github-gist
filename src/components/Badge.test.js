
import { render } from '@testing-library/react';

import Badge from './Badge';

describe('Badge', () => {
  it('should render correctly', async () => {
    const { container } = render(<Badge>my badge</Badge>);

    expect(container).toMatchSnapshot();
  });
});
