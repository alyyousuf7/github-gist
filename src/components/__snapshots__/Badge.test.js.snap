// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge should render correctly 1`] = `
<div>
  <div
    class="badge"
  >
    my badge
  </div>
</div>
`;
